app/serializers/move_serializer.rb

```diff
@@ -32,7 +32,9 @@ class MoveSerializer < ActiveModel::Serializer
     person.ethnicity
     person.gender
     from_location
+    from_location.suppliers
```

Comment:
we have locations in other serializers, can we add those there as well, eg allocations

Reply:
We're not using it currently in the frontend. I think we should be draconian in supporting only the includes that are needed.

Reply:
Although on second thoughts, I'm not sure. I think we should have chat about what includes we support separately. Going to ship this to get the frontend working and we can chat separately.